libs/wingsdk/src/target-sim/topic.inflight.ts

```diff
@@ -50,7 +50,7 @@ export class Topic
         timestamp: new Date().toISOString(),
       });
 
-      void (await fnClient.invoke(message).catch((err) => {
```

Comment:
Was this caught by eslint or did you just notice it? eslint does have a rule for this.

Reply:
Just noticed it. Definitely open to adding eslint rules though